### ISSUE TITLE
Feature questions - Bulk queries, API, data analytics

### ISSUE THREAD

**Issue by ianshefferman** [Netflix/Scumblr#4]
Hi, my team is interested in using Scumblr, but we have a few questions:

1. Are there plans to add support for bulk queries? For example, we may want to add a new search provider that will do a user search on Twitter. We may want to "follow" multiple users, and add them all in one bulk query, and then filter them as necessary on the results page. So basically some sort of class attribute that would make certain search providers turn the Query box into a multi-line text field, and do a separate search for each of those lines. On the search page it would ideally show up as one entry, but would just be a "multivalue search".

Depending on if my team agrees to go forward with this, I could possibly try and add that functionality myself and submit a pull request.

2. Is there a supported way for getting search results out of the database, perhaps with an API of some kind? Or plans for such a feature. I could throw up something like [Sandman](https://github.com/jeffknupp/sandman) or just write a script to pull data out of the sqlite database directly, but I'm wondering if you guys have any suggestions. An automated CSV export feature (appending all search results to a CSV each time a search completes) would probably do the job as well.

Basically, we'd like to use Scumblr more for the standardized data scraping and searching aspect, and then offload the results to a more powerful analytics platform (like Splunk) for further research.

Thanks.

**Comment by bensmith83** [comment]
I'd second this use case. Would be very useful to be able to see unions of hits from various keyword lists.
